packages/terra-functional-testing/src/terra-cli/express-server/expressServerHandler.js

```diff
@@ -7,6 +7,5 @@ module.exports = async (options) => {
     site: options.site,
   });
 
-  await server.createApp();
```

Comment:
I removed the call to createApp() because it is already being called inside [start()](https://github.com/cerner/terra-toolkit/blob/main/packages/terra-functional-testing/src/express-server/express-server.js#L48). This is also to match [AssetServerService](https://github.com/cerner/terra-toolkit/blob/main/packages/terra-functional-testing/src/services/wdio-asset-server-service.js#L39) where calling createApp() is not needed.

Reply:
I was gonna ask about that :)